plugins/kosync.koplugin/main.lua

```diff
@@ -25,12 +25,19 @@ local KOSync = InputContainer:new{
     title = _("Register/login to KOReader server"),
 }
 
+-- Creates a unique (hopefully) device id to identify this device. This function will only be
+-- called during the very first time KOSync has been initialized.
+local function createDeviceId()
+    return tostring(math.random(100,999))
```

Comment:
I have explained my thought in this review description. A UUID is 16 bytes, which is larger than a random number between [100, 999], which is three bytes only. And considering we distinct a device with the combination of user id, device model and this device id, a UUID or a random three digits number should have no difference. Since this UUID will need to be sent through network each time we sync a progress (both pull or push), these extra 13 bytes count.

But if you have a strong opinion, I can move this logic in to a more common place, and use a UUID instead.